src/extension/app.ts

```diff
@@ -317,11 +317,15 @@ class App {
             const useModifier = getBoolSetting(SETTINGS.USE_MODIFIER);
             this.isGrabbing = true;
 
-            if(validWindow(win) && !useModifier) {
-                activeMonitors().forEach(m => {
-                    this.tabManager[m.index]?.show();
-                });
-            }
+            if (!validWindow(win)) return;
```

Comment:
Hi @ViolentSempie! I fixed the modifiers-grabbing functionality that was broken for some reason. Are you ok with this change?